docs/index.html

```diff
@@ -17,5 +17,5 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/js-beautify/1.7.5/beautify.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/js-beautify/1.7.5/beautify-css.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/js-beautify/1.7.5/beautify-html.min.js"></script>
-<script type="text/javascript" src="runtime.26209474bfa8dc87a77c.js"></script><script type="text/javascript" src="polyfills.9d6088873d2a3c33ec93.js"></script><script type="text/javascript" src="scripts.3fdc5a8ed827a43484bd.js"></script><script type="text/javascript" src="main.5de2f82a9244e4324a18.js"></script></body>
+<script type="text/javascript" src="runtime.26209474bfa8dc87a77c.js"></script><script type="text/javascript" src="polyfills.9d6088873d2a3c33ec93.js"></script><script type="text/javascript" src="scripts.3fdc5a8ed827a43484bd.js"></script><script type="text/javascript" src="main.8fc24ee5787008133b5e.js"></script></body>
```

Comment:
format

Reply:
This is the file generated by Angular as part of the build process